src/table/components/TableBodyWrapper.jsx

```diff
@@ -8,90 +8,97 @@ import { getBodyCellStyle } from '../utils/styling-utils';
 import { bodyHandleKeyPress } from '../utils/handle-key-press';
 import { handleClickToFocusBody } from '../utils/handle-accessibility';
 
-function TableBodyWrapper({
-  rootElement,
-  tableData,
-  constraints,
-  selectionsAPI,
-  layout,
-  theme,
-  setShouldRefocus,
-  keyboard,
-  tableWrapperRef,
-  announce,
-}) {
-  const { rows, columns, paginationNeeded } = tableData;
-  const setFocusedCellCoord = useContextSelector(TableContext, (value) => value.setFocusedCellCoord);
-  const selectionDispatch = useContextSelector(TableContext, (value) => value.selectionDispatch);
-  // active: turn off interactions that affect the state of the visual representation including selection, zoom, scroll, etc.
-  // select: turn off selections.
-  const selectionsEnabled = !!selectionsAPI && !constraints.active && !constraints.select;
-  const columnRenderers = useMemo(
-    () => columns.map((column) => getCellRenderer(!!column.stylingInfo.length, selectionsEnabled)),
-    [columns.length, selectionsEnabled]
-  );
-  const bodyCellStyle = useMemo(() => getBodyCellStyle(layout, theme), [layout, theme.name()]);
-  const hoverEffect = layout.components?.[0]?.content?.hoverEffect;
-  const cellStyle = { color: bodyCellStyle.color, backgroundColor: theme.table.backgroundColor };
+const TableBodyWrapper = forwardRef(
```

Comment:
To fix a warning, now it looks weird
Reference:
https://stackoverflow.com/questions/59716140/using-forwardref-with-proptypes-and-eslint
https://github.com/facebook/react/issues/16653